frontend/src/reducer/notificationPage.js

```diff
@@ -146,7 +146,7 @@ export default function notificationPage (state = defaultNotificationsObject, ac
       }
     }
 
-    case `${READ}/${NOTIFICATION_LIST}`: {
+    case `${READ}/${EVERY_NOTIFICATION}`: {
```

Comment:
```suggestion
    case `${READ}/${ALL_NOTIFICATIONS}`: {
```
Just a suggestion, you don't have to apply if you prefer the current name: since in most of the code we use ALL and not EVERY. But if you apply, it would be good to change on `readEveryNotifications` too. If you decide not to apply it, it would still be good to normalize the plural, because here we have `EVERY_NOTIFICATION` without s and in the `readEveryNotifications` function with s.

Reply:
I feel like ``every`` is more adapted here since we are reading every one of the notifications.
For the ``s`` it's after a debate with Skylsmoi. His first review was ``EVERY_NOTIFICATIONS`` to ``EVERY_NOTIFICATION_LIST``. However this key (``EVERY_NOTIFICATIONS`) is meant to say _We are reading every notifications_. Since the ``s`` is confusing (see Skylsmoi) we've decided to remove the plural.

Reply:
I think the naming should be decided in ordre to improve thé understandability. If you process a liste, a plural form is a better naming choice, I think.

A naming convention should fixe this (what is the rule for plural form var naming)

Reply:
I'm not going to lie, I'm very confused about this..
So I've done some researches and here is what I've got:
- ``all`` refers to the whole group
- ``Every`` focuses on each individual member of the whole group

With all -> plural
With every -> singular

_So, in every case ``every_notifications`` is wrong grammatically speaking_

I can change to ``all_notification_list`` (with ``list`` and not just plural because the code is using this convention in the frontend) but as I said, the key to redux here is to read _every_ notification. And put it as read _all_ notifications confuse me as possible is human possible since we are speaking of each notifications that aren't read already

Reply:
What I say is *convention* is better than *grammar* for code maintenance.

If the rule applied is different, when maintaining code I will ask myself : "why is the naming différent? Is there a specific behavior?"

I juste reacted to the discussion about this is more correct vs the code is full or "all".

Conventions may change. I have no opinion here.

Reply:
I mean:

- same behavior -> same naming rule
- different behavior -> different naming rule